app/views/miq_ae_customization/_dialog_field_form.html.haml

```diff
@@ -48,6 +48,30 @@
           &nbsp;
           = _('Only 1 Date or Date/Time element can be present in a Dialog')
 
+    .form-group
+      %label.col-md-2.control-label
+        = _('Auto refresh')
+      .col-md-8
+        = check_box_tag('field_auto_refresh', '1',
+                        @edit[:field_auto_refresh],
+                        'data-miq_observe_checkbox' => {:url => url}.to_json)
+
+    .form-group
+      %label.col-md-2.control-label
+        = _('Auto Refresh other fields when modified')
+      .col-md-8
+        = check_box_tag('field_trigger_auto_refresh', '1',
+                        @edit[:field_trigger_auto_refresh],
+                        'data-miq_observe_checkbox' => {:url => url}.to_json)
```

Comment:
This I think is fine to be in this file, since it pertains to all types of dialog fields, but it should probably be moved into the options section, right now it's in the "Element information" section.